### PR TITLE
feat: :sparkles: Show cookie at the bottom at the screen

### LINKED ISSUE #5: Create component: Cookie Disclaimer

### PR DESCRIPTION
**Pull request by MatsHaby**
The cookie will show up when loading the page and will be removed the clicking the button: Accept

### CODE REVIEW

**Comment by dmlfahfn** [comment]
ser bra ut!

**Comment by josefinelofgren** [comment]
Både ser ut och fungerar som den ska, snyggt!

File: design/src/App.scss [hunk 1]
```diff
@@ -77,6 +77,48 @@ $hover-color: #f8cc80;
...
+    color: #ffffff;
+    min-height: 212px;
+    width: 100%;
+    position: fixed;
+    top: auto;
+    bottom: 0;
+    align-items: center;
+    justify-content: space-between;
+    padding: 30px 30px 30px 30px;
+    &__button {
```
Comment (postmodernistx) [inline design/src/App.scss:93]: Nice 👍

File: design/src/App.scss [hunk 2]
```diff
@@ -125,4 +175,16 @@ $hover-color: #f8cc80;
             font-size: 26px;
         }
     }
+
+    .cookie {
```
Comment (postmodernistx) [inline design/src/App.scss:179]: Super clean and easy-to-read code!

**Review by postmodernistx — APPROVED** [review]
Bonus points for gitmoji and the categorization of the PR! ❤️ 

Super clean code!

Next level pro tip: add a screenshot of the changes :)

**Pull request merged**